terraform/modules/dandiset_bucket/main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_s3_bucket" "dandiset_bucket" {
+  # provider = var.aws_provider
+
+  bucket = var.bucket_name
+  // Public access is granted via a bucket policy, not a canned ACL
+  acl = "private"
+
+  cors_rule {
```

Comment:
This is only necessary if you want web browsers to be able to make Ajax operations on this bucket. If the access it just from server backends, direct downloads, or non-browser clients, you don't need CORS.